indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,128 @@
+(function(global) {
+    'use strict';
+
+    function handleRowSelection() {
+        $('.sessions-wrapper').on('change', 'table.i-table input.select-row', function() {
+            $(this).closest('tr').toggleClass('selected', this.checked);
+            $('.js-requires-selected-row').toggleClass('disabled', !$('.sessions input:checkbox:checked').length)
+        }).trigger('change');
+    }
+
+    function setupTableSorter() {
+        $('.sessions .tablesorter').tablesorter({
+            cssAsc: 'header-sort-asc',
+            cssDesc: 'header-sort-desc',
+            cssInfoBlock: 'avoid-sort',
+            cssChildRow: 'session-blocks-row',
+            headerTemplate: '',
+            sortList: [[1, 0]]
+        });
+    }
+
+    function setupColorSwitches() {
+        $('.js-color-switch').map(function() {
```

Comment:
`.each(..)` - `map()` is used when you care about the return value of the callback